src/components/admin/root-security-group.ts

```diff
@@ -0,0 +1,6 @@
+export class RootSecurityGroup {
+  canReadOrgs = true;
+  canListUsers = true;
+  canCreateUser = true;
+  canCreateSecurityGroup = true;
+}
```

Comment:
We are trying to avoid using classes for DTOs / value objects. This keeps logic in services and allows us to spread properties around without fear of loosing an unspoken requirement of the object prototype.

If both type and default value is needed here's the pattern we use in this rest of the codebase.
```suggestion
export class RootSecurityGroup {
  static defaultValue = new RootSecurityGroup();
  protected constructor() {}

  canReadOrgs = true;
  canListUsers = true;
  canCreateUser = true;
  canCreateSecurityGroup = true;
}
```

If the type isn't really needed just make it a constant:
```suggestion
export const RootSecurityGroup = {
  canReadOrgs: true,
  canListUsers: true,
  canCreateUser: true,
  canCreateSecurityGroup: true,
};
```

Reply:
10-4, going with the `const` approach